markdown/h2m/handlers/cards.ts

```diff
@@ -1,13 +1,26 @@
+import * as Gettext from "node-gettext";
 import { h } from "../h";
 import { asArray } from "../utils";
 import { toText } from "./to-text";
 import { QueryAndTransform } from "./utils";
 
+const getTextDefaultDomainName = "messages";
+
+function initLocalizations(locale) {
```

Comment:
Could we initialize these only once? Either lazy of all locales at once.

Reply:
I was not fully sure of what you expected and I went with https://github.com/mdn/yari/pull/4251/commits/e9cccd723a9793471befb16730d23c24d7ab926e, let me know if it's OK.